app/assets/javascripts/components/client/RegisterClient.jsx

```diff
@@ -0,0 +1,183 @@
+class RegisterClient extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {success: null};
```

Comment:
construct `this.state` in a more generalized format in case more states are added in the future.
`this.state = {
        success: null,
}`